agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -3439,7 +3439,7 @@ bool backhaul_manager::get_media_type(const std::string &interface_name,
                     if ((std::get<0>(tuple) == radio_info.frequency_band) &&
                         (std::get<1>(tuple) == radio_info.max_bandwidth)) {
                         media_type = std::get<2>(tuple);
-
+                        result     = true;
```

Comment:
Why can't we just return true in these cases?

Reply:
It's the multiple exit points vs. single exit point discussion, which I'm not sure we put to a vote.